src/tmt_web/generators/json_generator.py

```diff
@@ -7,7 +7,7 @@
 
 def _create_json_data(obj: Test | Plan, logger: tmt.Logger) -> dict[str, Any]:
     """
-    Helper function to create the JSON data from a test or plan object
+    Helper function to create the JSON data from a test or plan object.
```

Comment:
Should enable `D` ruff rules for these.

Reply:
That's probably a comment to the https://github.com/teemtee/web/pull/4 or separate PR entirely imho.